spec/models/user_spec.rb

```diff
@@ -0,0 +1,20 @@
+require 'rails_helper'
+
+RSpec.describe User, type: :model do
+  describe "schema" do
+    it { should have_db_column(:email).of_type(:string).with_options(null: false) }
+    it { should have_db_column(:encrypted_password).of_type(:string) }
+
+    it { should have_db_column(:updated_at) }
+    it { should have_db_column(:created_at) }
```

Comment:
Generally, yes. According to shoulda matchers, though, this should still work regardless of how rspec is configured. We could do `is_expected.to` as an alternative.